src/DataWrangling/metadata.jl

```diff
@@ -2,7 +2,7 @@ using CFTime
 using Dates
 using Base: @propagate_inbounds
 
-struct Metadata{D, V}
+struct Metadata{V, D}
     name  :: Symbol
     dates :: D
     dataset :: V
```

Comment:
```suggestion
    dataset :: V
    dates :: D
```

you might as well also do this to reduce the possibility of confusion